src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/AbstractIssue.java

```diff
@@ -71,6 +77,18 @@
 	@Temporal(TemporalType.TIMESTAMP)
 	private Date closed;
 
+	// Left away @NotEmpty for now, since existing tests should be updated for it to work
```

Comment:
Ah yes... That is indeed a problem. You could go through the hassle of writing your own Hibernate value **generator**. But I have a work around for now:

```java
pullRequest.setTitle(branchName);
pullRequests.persist(pullRequest);
pullrequest.setTitle(computeTitle...);
```